docs/integrations/sources/zendesk-support.md

```diff
@@ -70,7 +71,8 @@ The Zendesk connector ideally should not run into Zendesk API limitations under
 
 | Version  | Date       | Pull Request                                             | Subject                                                                                                                                                                                                                            |
 | :------- | :--------- | :------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `0.3.1`  | 2023-06-02 | [26945](https://github.com/airbytehq/airbyte/pull/26945) | Make `Ticket Metrics` stream to use cursor pagination  |
+| `0.4.0`  | 2023-06-09 | [27156](https://github.com/airbytehq/airbyte/pull/27156) | Add stream `Posts`                                                                                                                                                                                                                 |
```

Comment:
If this is backwards compatible, it can just be a minor version change.

```suggestion
| `0.3.2`  | 2023-06-09 | [27156](https://github.com/airbytehq/airbyte/pull/27156) | Add stream `Posts`                                                                                                                                                                                                                 |
```